### DISCUSSION TITLE
Tab bar indication only for stack layout

### DISCUSSION THREAD

**Discussion by r-darwish** [kovidgoyal/kitty#6563] [Q&A] [answered]
I'm using `map ctrl+alt+z toggle_layout stack` a few times a day in my workflow. Before Kitty I used to work mostly in Tmux, and my tab bar was configured so that a 🔍 symbol would be shown when using the Tmux equivalent of the Kitty's stack layout, otherwise it's easy to forget that you have multiple windows attached to this tab.

Is there any way to show an indication in the tab bar only when the stack layout is active?

**Comment by kovidgoyal** [comment]
See tab_title_template which has a variable named layout_name

**Reply by kovidgoyal** [reply, marked as answer]
For completeness::

```
tab_title_template {"X " if layout_name == "stack" else ""}{title}

**Reply by r-darwish** [reply]
This level of customization that Kitty offers is amazing, thank you. I ended up doing this:
```
tab_title_template "{bell_symbol}{activity_symbol}{'  ' if layout_name == 'stack' and num_windows > 1 else ''}{fmt.fg.tab}{index}. {title}"
```
